src/battle.ts

```diff
@@ -1219,6 +1213,19 @@ class Battle {
 			let pp = (target && target.side !== pokemon.side && toId(target.ability) === 'pressure' ? 2 : 1);
 			pokemon.rememberMove(moveName, pp);
 		}
+		pokemon.lastMove = move.id;
+		this.lastMove = move.id;
+		if (move.id === 'wish' || move.id === 'healingwish') {
+			pokemon.side.wisher = pokemon;
+		}
+        }
```

Comment:
>:(

Reply:
Oops, I'd changed my editor settings to work on another project and forgot to check them. Sorry about that...